Dockerfile

```diff
@@ -48,6 +48,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
       python3-pip                    \
       bash-completion                \
       vim                            \
+      docker                         \
```

Comment:
1. AFAIK, the `docker` package in debian or ubuntu is not the Linux container package that you want.
2. Only installing docker cli is better than all since you do not need the daemon

You can just copy from the `docker:dind` image to install docker cli:

 ```
COPY --from=docker:dind /usr/local/bin/docker /usr/local/bin/
```

Or extract it from the release file manually, https://download.docker.com/linux/static/stable/x86_64